lib/_debug_agent.js

```diff
@@ -185,6 +195,11 @@ Client.prototype.onCommand = function onCommand(cmd) {
   this.binding.sendCommand(cmd.body);
 
   this.agent.notifyWait();
+  
+  if (cmd.body == disconnectRequest) {
```

Comment:
Wouldn't it be better to `JSON.parse` the command and check it's type? It could be guarded by RegExp match if you are afraid of hurting performance.